draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -676,9 +676,42 @@
 
 		<section anchor="sec_determining_values" title="Determining values to be advertised in the Flooding Parameters TLV">
 			<t>The values that a receiver advertises do not need to be perfect. If the values are too low then the transmitter will not use the full bandwidth or available CPU resources. If the values are too high then the receiver may drop some LSPs during the first RTT and this loss will reduce the usable receive window and the protocol mechanisms will allow the adjacency to recover. Flooding slower than both nodes can support will hurt performance, as will consistently overloading the receiver.</t>
+
+			<section anchor="sec_determining_values_static" title="Static values">
 			<t>The values advertised need not be dynamic as feedback is provided by the acknowledgment of LSPs in SNP messages. Acknowledgments provide a feedback loop on how fast the LSPs are processed by the receiver. They also signal that the LSPs can be removed from receive window, explicitly signaling to the sender that more LSPs may be sent. By advertising relatively static parameters, we expect to produce overall flooding behavior similar to what might be achieved by manually configuring per-interface LSP rate-limiting on all interfaces in the network. The advertised values could be based, for example, on offline tests of the overall LSP-processing speed for a particular set of hardware and the number of interfaces configured for IS-IS. With such a formula, the values advertised in the Flooding Parameters TLV would only change when additional IS-IS interfaces are configured.</t>

```

Comment:
The last sentence of this paragraph seems inaccurate and not necessary. I suggest it be removed.
I say "inaccurate" because you go on in the subsequent paragraphs to discuss (quite correctly) that other control plane traffic could also impact the values.

Reply:
This sentence is not new and not coming from me. In the context of this section, which explains that this works with static value, I think that the sentence is good enough given the specific context set just before (IS-IS process is capable of processing a specific amount of LSP per secondes. So a simple way is to split this capability per neighbhor).